CONTRIBUTING.md

```diff
@@ -27,15 +28,22 @@ We also recommend you to learn about KubeVela's [design](docs/en/design.md) befo
 git clone git@github.com:oam-dev/kubevela.git
 ```
 
+KubeVela includes two parts, `vela cli` and `vela core`.
+
+- The `vela cli` is a command line tool that can build, run apps(with the help of `vela core`).
```

Comment:
We can introduce `vela core` first as `vela cli` is referring to `vela core`

Reply:
yes thanks